recipes/doxygen/all/conanfile.py

```diff
@@ -11,22 +11,24 @@ class DoxygenInstallerConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/doxygen/doxygen"
     license = "GPL-2.0-or-later"
+    settings = "os", "arch", "compiler"
 
-    settings = "os", "arch",
-
-    def config(self):
+    def configure(self):
         if self.settings.os in ["Linux", "Macos"] and self.settings.arch == "x86":
             raise ConanInvalidConfiguration("Doxygen is not supported on {}-{}".format(self.os, self.arch))
+        if tools.Version(self.version) >= "1.8.18" and self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "6":
+            raise ConanInvalidConfiguration("Doxygen requires GLIBCXX_3.4.22")
```

Comment:
The error
```
doxygen: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.22' not found
```
has nothing to do with the compiler or its compiler version.

What this means is that the downloaded executable is linked against a newer [GNU C library glibc](https://www.gnu.org/software/libc/). Some build bots have an older library installed, which gives rise to the error above.

As of now, conan does not include the glibc version in the package id.

Reply:
https://gcc.gnu.org/onlinedocs/libstdc++/manual/abi.html I am not sure what you mean by it "has nothing to do with compiler version" it's a direct relationship but there is a connection...

I release that so it's just a question of picking a bot to package this tool. However I was expecting feedback about this. I am not familiar with any best practice for packaging tools.

Reply:
Oh sorry, so now we have a problem with `GLIBC` and with `GLIBCXX`...

Reply:
Look like it! Next time I'm at my desk ill google for it